crates/criteria_policy_base/README.md

```diff
@@ -0,0 +1,132 @@
+This is a helper crate that allows to easily scaffold a policy that validates a
+list of strings using a well defined list of criteria.
+
+The allowed criteria are:
+
+- `containsAnyOf`
+- `doesNotContainAllOf`
+- `containsAllOf`
+- `doesNotContainAllOf`
+- `containsOtherThan`
+- `doesNotContainOtherThan`
+
+Right now we expect this library to be consumed by policies that are validating
+environment variables, labels and annotations.
+
+## Rules operators logic tables
+
+These are some tables to help you understand the logic of the operators:
+
+### `containsAnyOf`
+
+Given these `environmentVariables` settings: `[a, b]`
```

Comment:
Shouldn't we update the documentation to be more generic? Now, this will not be used only for environment variable any more.

Reply:
It's a good point, yet I think it provides an understandable example tbf. We do preface it with:
> Right now we expect this library to be consumed by policies that are validating
environment variables, labels and annotations.